### ISSUE TITLE
get colliding link with insertRobotIntoMapSelfCollAware()

### ISSUE THREAD

**Issue by giorgionicola** [fzi-forschungszentrum-informatik/gpu-voxels#125]
I am using the method gpu_voxels::insertRobotIntoMapSelfCollAware() method to get the collision between two robots, the outputs is only a boolean.

Is there a way to know also which is the couple of colliding links?

**Comment by cjue** [comment]
@giorgionicola: Yes, you can use the optional parameter `colliding_meanings` to collect that information.

The description can be found in the header file with the definition of `GpuVoxels::insertRobotIntoMapSelfCollAware()`:  
https://github.com/fzi-forschungszentrum-informatik/gpu-voxels/blob/master/packages/gpu_voxels/src/gpu_voxels/GpuVoxels.h#L318

Implementation: https://github.com/fzi-forschungszentrum-informatik/gpu-voxels/blob/master/packages/gpu_voxels/src/gpu_voxels/voxelmap/BitVoxelMap.hpp#L339

Example: https://github.com/fzi-forschungszentrum-informatik/gpu-voxels/blob/master/packages/gpu_voxels/src/examples/URDF_Loader_Selfcoll.cpp#L192

The example code linked to above needs some updates from our internal gpu_voxels repo, but the general usage of the API is correct.

Let me know if you encounter further obstacles.
